src/python/WMCore/ReqMgr/CherryPyThreads/StatusChangeTasks.py

```diff
@@ -124,8 +95,6 @@ def advanceStatus(self, config):
         wfStatusDict = gqService.getWorkflowStatusFromWQE()
 
         self.logger.info("Advancing statuses")
-        if getattr(config, "enableMSStatusTransition", False):
```

Comment:
Does this config is used anywhere else in WMCore code. If yes, please check how it is used, if no, can you provide changes to configuration to remove it completely?

Reply:
No, it's only used by this cherrypy thread. Configuration changes have been provided in the initial description of this PR.

Reply:
ok, then I think changes are fine.